app/controllers/ask_trn_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+class AskTrnController < ApplicationController
+  include EnforceQuestionOrder
+
+  layout "two_thirds"
+
+  def new
+    @ask_trn_form = AskTrnForm.new(trn_request:)
+    @ask_trn_form.do_you_know_your_trn = "true" if @ask_trn_form
+      .trn_request
+      .trn_from_user
+      .present?
```

Comment:
Could this logic sit in the initializer for `AskTrnForm`?

Reply:
@tvararu not sure as it is not an attribute of the `trn_request` model. What do you suggest?